docs/manual.md

```diff
@@ -476,3 +476,18 @@ following transactions:
 Support for more complex inline math would require substantial changes
 to the parser.
 
+
+Bugs and Contributions
+----------------------
```

Comment:
quick one on this (I haven't read the rest yet, sorry!)

There is a convention of using a dedicated `docs/CONTRIBUTING.md` file to describe contribution guidelines. If it's available, [GitHub points to it](https://opensource.guide/starting-a-project/) when someone files a pull-request, which is nice.